### PR TITLE
Add Localization guide

### LINKED ISSUE #19: Localization Guide
Two parts:

POEditor and how to sign up / use it.

In-game localizing (installing Python 3, running tokenizer.py, etc)

### PR DESCRIPTION
**Pull request by Gocnak**
Closes #19

### CODE REVIEW

File: _posts/guides/2022-10-11-guide_localization.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+### Testing Strings from POEditor
+
+There's a helpful Python script to pull in the latest translation work, found inside `Momentum Mod Playtest/momentum/tools/`, called `get_poeditor_strings.py`.
+
+This script requires [the latest version of Python 3](https://www.python.org/downloads/), and [a readonly POEditor API token](https://poeditor.com/account/api) to be used correctly.
+
+To run this script, open a terminal and set the following variables:
+
+On Windows (PowerShell):
```
Comment (tsa96) [inline _posts/guides/2022-10-11-guide_localization.md:81]: I find it's most convenient to add this to environment variables in the Windows Control Panel so they stick around. So can you add

```
Alternatively, you can go to System Properties > Environment Variables and add to your User Variables there.
```

File: _posts/guides/2022-10-11-guide_localization.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+```powershell
+$Env:POEDITOR_PROJECT_ID = "156379"
+$Env:POEDITOR_API_KEY = "(your read-only API key)"
+```
+
+On Linux:
```
Comment (tsa96) [inline _posts/guides/2022-10-11-guide_localization.md:87]: Add something like

```
On Linux (it might help to put this in your .bashrc file or similar
```

**Review by tsa96 — CHANGES_REQUESTED** [review]
This is great, thanks for doing this. It all reads correctly, only suggestion I have is adding some more convenient env var handling tips

**Comment by hexaflexahexagon** [comment]
Closes https://github.com/momentum-mod/docs/issues/19 ? @Gocnak

**Review by tsa96 — APPROVED** [review]

**Pull request merged**